### DISCUSSION TITLE
Is there any faucet to play around with?

### DISCUSSION THREAD

**Discussion by AnakDagang** [Chia-Network/chia-blockchain#497] [Q&A] [answered]
I believe I have successfully plot and farm on version 0.1.17 but it will take me around a year to find a block and wonder if there's a faucet I could try to receive XCH

**Comment by hoffmang9** [comment, marked as answer]
If you put your testnet wallet receive address here or in the #testnet channel on our Keybase discussion, people will happily send you TXCH. However, the only way to get XCH is going to be by farming or by purchasing it in the open market once it starts trading. https://keybase.io/team/chia_network.public

You should note that your odds of winning an XCH when farming are going up by 16x with the new consensus that will be on testnet shortly.

**Reply by AnakDagang** [reply]
sorry I mean TXCH here's the address txch1z4syhrpxznkvwph6xuqreg5gxm8qs7tuem0mf6x5e65lay2zngpqjdgk4u

**Reply by hoffmang9** [reply]
Inbound

**Comment by AnakDagang** [comment]
just FYI the 100 TXCH you sent me got lost when I format my PC and reinstall the wallet although I did key-in the saved the 24 word seed

probably a bug? 

![Capture](https://user-images.githubusercontent.com/66496073/97805951-18b22e00-1c94-11eb-8b69-022bd4e27a83.PNG)

**Reply by dorkmo** [reply]
did your node and wallet sync already?

**Reply by AnakDagang** [reply]
> did your node and wallet sync already?

yes i even plot successfully still 0 balance TXCH

**Comment by alex-shuhman** [comment]
You may use my wrapper for testnet faucet here http://chia-faucet.com
